dotnet/src/InternalUtilities/process/Runtime/ProcessEvent.cs

```diff
@@ -8,10 +8,13 @@ namespace Microsoft.SemanticKernel.Process.Runtime;
 /// </summary>
 /// <param name="Namespace">The namespace of the event.</param>
 /// <param name="InnerEvent">The instance of <see cref="KernelProcessEvent"/> that this <see cref="ProcessEvent"/> came from.</param>
+/// <param name="IsError">This event represents a runtime error / exception raised internally by the framework.</param>
 [DataContract]
+[KnownType(typeof(KernelProcessError))]
 public record ProcessEvent(
     [property: DataMember] string? Namespace,
```

Comment:
not your change but should this just be `string` checked in the code, there seems to always be a namespace.

Reply:
This is addressing the `KernelProcessEvent.Data` property (which may be of type `KernelProcessError`.  You are certainly astute to zoom in and notice that something may not be ideal here...it certainly isn't.  We have a larger issue with _data-contracts_ and _known-types_ that I'm currently looking at in a different branch.  This dosn't solve the larger issue, but it doesn't move us backwards either...and enables the current change.  Once we have solution for the broader issue, I'd expect this to go away.